test/inspect.test.js

```diff
@@ -41,14 +41,14 @@ deepTestFolders.forEach( function(folder) {
 
 tap.test('php plugin for project with many deps', function (t) {
   var projFolder = './test/stubs/many_deps_php_project';
-  return plugin.inspect(projFolder, 'composer.lock', options)
+  return plugin.inspect(projFolder, './composer.lock', options)
```

Comment:
maybe also keep the previous test so we know we're supporting both formats?

Reply:
The previous test uses the simple file name form. I was looking for the smallest change that surfaces the bug in the current implementation.

Reply:
Previous === the test above this one.